larpandoracontent/LArObjects/LArCaloHit.h

```diff
@@ -294,6 +347,8 @@ inline pandora::StatusCode LArCaloHitFactory::Read(Parameters &parameters, pando
     LArCaloHitParameters &larCaloHitParameters(dynamic_cast<LArCaloHitParameters &>(parameters));
     larCaloHitParameters.m_larTPCVolumeId = larTPCVolumeId;
     larCaloHitParameters.m_daughterVolumeId = daughterVolumeId;
+    larCaloHitParameters.m_hitScores = hitScores;
+    larCaloHitParameters.m_hitScoreLabels = hitScoreLabels;
 
```

Comment:
There's a need for the corresponding Write function to be updated to conform to what's expected in the Read function above.